kickstarts/base.ks.erb

```diff
@@ -73,6 +73,12 @@ do
   grep -q 'DEVICE=' $FILE || echo "DEVICE=$DEVICE" >> $FILE
 done
 
+# Disable zeroconfig to allow access to meta-data service by cloud-init
+cat > /etc/sysconfig/network << EOF
```

Comment:
is this a create (>) or append ( >>), i.e. will file never have anything else there ?